tests/test_plugin_loader.py

```
@@ -2352,7 +2813,9 @@ def test_run_type_error_non_string_args(self):
     def test_run_value_error_shell_true(self):
         """Test _run raises ValueError for shell=True."""
         with self.assertRaises(ValueError) as cm:
-            _run(["git", "status"], shell=True)  # noqa: S604 - intentional for test
+            _run(
```

Ruff (0.15.7)

[error] Function call with `shell=True` parameter identified, security issue

(S604)